README.md

```diff
@@ -9,6 +9,12 @@ UI for Inter Protocol PSM
 By default it it will connect to the prod Wallet UI. To connect to a local one, use
 http://127.0.0.1:5173/?wallet=local .
 
+## Contributing
+
+For bugs and feature requests, open a [new issue](https://github.com/Agoric/dapp-psm/issues/new).
+
+For PRs, develop against the [main](https://github.com/Agoric/dapp-psm/tree/main) branch.
+
```

Comment:
Looks good, maybe on L22:


> To deploy, push to that branch. e.g. if you've qualified main,

To deploy your changes after they've been merged into `main`: